test/common/stats/thread_local_store_speed_test.cc

```diff
@@ -45,7 +45,7 @@ class ThreadLocalStorePerf {
 
   void accessCounters() {
     for (auto& stat_name_storage : stat_names_) {
-      store_.counterFromStatName(stat_name_storage->statName());
+      store_.Stats::Store::counterFromStatName(stat_name_storage->statName());
```

Comment:
I think this hack is due to having virtual/non-virtual overloads of the same name.

Reply:
yeah, ill rename one of them to get rid of this.